TeamSnapSDK/SDK/DataTypes/TSDKUser.m

```diff
@@ -202,4 +202,22 @@ + (NSURL * _Nullable)persistenceFilePathWithParentObject:(TSDKCollectionObject *
     return [[[self persistenceBaseFilePath] URLByAppendingPathComponent:[self SDKType]] URLByAppendingPathComponent:@"current"];
 }
 
+- (TSDKHighestRoleType)highestRoleKey {
```

Comment:
@SkylerSeamans Don't we already have something similar on Role/TeamRole that we leverage for the `isAtLeastPermission:` API?

Reply:
@larsacus TeamRole is member/team centric. I believe this is at the user-level. @jrahaim can correct if that's wrong.

Reply:
@larsacus  Kinda, sorta.. Those are based on booleans from the API (not strings) and separate from the users highest role. it would require having the users entire Team Matrix loaded in order to calculate.

Reply:
bah. ok.

Reply:
And, what Skyler said about team specific vs user overall.